src/activ_api.cpp

```diff
@@ -69,31 +69,36 @@ extern "C" miopenStatus_t miopenGetActivationDescriptor(miopenActivationDescript
     });
 }
 
-static void LogCmdActivation(const miopenTensorDescriptor_t xDesc,
-                             const miopenActivationDescriptor_t activDesc,
-                             const bool Fwd)
+void LogCmdActivation(const miopen::TensorDescriptor& x_desc,
```

Comment:
[R] Move this into `miopen::debug` namespace.

Reply:
Is this needed? I'd like to merge this sooner. It wasn't in the namespace originally and I'd rather limit changes to what is necessary for next steps.

Reply:
changed it

Reply:
[resolved] Thanks